terraform/gcp/cloud-init.sh.tpl

```diff
@@ -0,0 +1 @@
+# add conf here if needed
```

Comment:
:thinking:  I'm ok with that solution.
But while reviewing I was wondering if it could be a good idea or not to put is as part of a stackform textarea variable.

I mean we could put this raw text as part of one terraform variable like var.user-data-sh
Then in the cloud-init.sh.tpl simply render as content ${var.user-data-sh}

This will let the user inject and handle user data scripts directly through stackforms

Reply:
(Maybe we could discuss about it internally with the others devops to see what will be the best)